pkg/apis/k8s.cni.cncf.io/v1beta2/types.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta2
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why do you need the changes? I suppose we don't need that.

Reply:
hmmm, I use VS Code, when I save the file, it invokes some tools(probably gofmt) to do post-processing, which cause this change.

Reply:
removed this change, PTAL, thanks